customLabels.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"encoding/json"
+	go_cache "github.com/patrickmn/go-cache"
+	"strings"
+	"time"
+	plog "github.com/prometheus/common/log"
+)
+
+// Custom labels for ConsumerGroups
+type CustomCGLagLabels struct {
+	labelByPrefix					map[string]string
+	labelCache					*go_cache.Cache
+	cacheExpirationInMin				time.Duration
+	cachecleanupIntervalinMin			time.Duration
+}
+
+type Notifier map[string]map[string][]string
+
+func NewCustomCGLagLabels(config string, CacheExpirationInMin, CachecleanupIntervalinMin time.Duration) (*CustomCGLagLabels, error){
```

Comment:
what do you think of adding the type for `CacheExpirationInMin` for consistency ?

Reply:
when the types are the same in go you can define it at the end, this is totally fine, there is not need to do it for each parameter https://tour.golang.org/basics/5